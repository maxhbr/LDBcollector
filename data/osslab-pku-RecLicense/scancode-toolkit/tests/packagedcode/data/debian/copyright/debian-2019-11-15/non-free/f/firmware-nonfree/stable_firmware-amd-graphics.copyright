Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
Upstream-Name: linux-firmware
Upstream-Contact: linux-firmware@kernel.org
Source: https://git.kernel.org/cgit/linux/kernel/git/firmware/linux-firmware.git

Files: r128/r128_cce.bin, radeon/*_cp.bin,
 radeon/R600_me.bin, radeon/R600_pfp.bin,
 radeon/RS780_*.bin, radeon/RV*_me.bin,
 radeon/RV*_pfp.bin
Copyright: 2000-2009, Advanced Micro Devices, Inc.
License: Binary redistribution (AMD permissive)
 All Rights Reserved.
 .
 Permission is hereby granted, free of charge, to any person obtaining a
 copy of this software and associated documentation files (the "Software"),
 to deal in the Software without restriction, including without limitation
 the rights to use, copy, modify, merge, publish, distribute, sublicense,
 and/or sell copies of the Software, and to permit persons to whom the
 Software is furnished to do so, subject to the following conditions:
 .
 The above copyright notice and this permission notice (including the next
 paragraph) shall be included in all copies or substantial portions of the
 Software.
 .
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
 IN NO EVENT SHALL THE COPYRIGHT OWNER(S) AND/OR ITS SUPPLIERS BE
 LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.

Files: amdgpu/*.bin, radeon/*.bin
Copyright: 2009-2016, Advanced Micro Devices, Inc.
License: Binary redistribution (AMD restrictive)
 All rights reserved.
 .
 REDISTRIBUTION: Permission is hereby granted, free of any license fees,
 to any person obtaining a copy of this microcode (the "Software"), to
 install, reproduce, copy and distribute copies, in binary form only, of
 the Software and to permit persons to whom the Software is provided to
 do the same, provided that the following conditions are met:
 .
 No reverse engineering, decompilation, or disassembly of this Software
 is permitted.
 .
 Redistributions must reproduce the above copyright notice, this
 permission notice, and the following disclaimers and notices in the
 Software documentation and/or other materials provided with the
 Software.
 .
 DISCLAIMER: THE USE OF THE SOFTWARE IS AT YOUR SOLE RISK.  THE SOFTWARE
 IS PROVIDED "AS IS" AND WITHOUT WARRANTY OF ANY KIND AND COPYRIGHT
 HOLDER AND ITS LICENSORS EXPRESSLY DISCLAIM ALL WARRANTIES, EXPRESS AND
 IMPLIED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
 MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT.
 COPYRIGHT HOLDER AND ITS LICENSORS DO NOT WARRANT THAT THE SOFTWARE WILL
 MEET YOUR REQUIREMENTS, OR THAT THE OPERATION OF THE SOFTWARE WILL BE
 UNINTERRUPTED OR ERROR-FREE.  THE ENTIRE RISK ASSOCIATED WITH THE USE OF
 THE SOFTWARE IS ASSUMED BY YOU.  FURTHERMORE, COPYRIGHT HOLDER AND ITS
 LICENSORS DO NOT WARRANT OR MAKE ANY REPRESENTATIONS REGARDING THE USE
 OR THE RESULTS OF THE USE OF THE SOFTWARE IN TERMS OF ITS CORRECTNESS,
 ACCURACY, RELIABILITY, CURRENTNESS, OR OTHERWISE.
 .
 DISCLAIMER: UNDER NO CIRCUMSTANCES INCLUDING NEGLIGENCE, SHALL COPYRIGHT
 HOLDER AND ITS LICENSORS OR ITS DIRECTORS, OFFICERS, EMPLOYEES OR AGENTS
 ("AUTHORIZED REPRESENTATIVES") BE LIABLE FOR ANY INCIDENTAL, INDIRECT,
 SPECIAL OR CONSEQUENTIAL DAMAGES (INCLUDING DAMAGES FOR LOSS OF BUSINESS
 PROFITS, BUSINESS INTERRUPTION, LOSS OF BUSINESS INFORMATION, AND THE
 LIKE) ARISING OUT OF THE USE, MISUSE OR INABILITY TO USE THE SOFTWARE,
 BREACH OR DEFAULT, INCLUDING THOSE ARISING FROM INFRINGEMENT OR ALLEGED
 INFRINGEMENT OF ANY PATENT, TRADEMARK, COPYRIGHT OR OTHER INTELLECTUAL
 PROPERTY RIGHT EVEN IF COPYRIGHT HOLDER AND ITS AUTHORIZED
 REPRESENTATIVES HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.  IN
 NO EVENT SHALL COPYRIGHT HOLDER OR ITS AUTHORIZED REPRESENTATIVES TOTAL
 LIABILITY FOR ALL DAMAGES, LOSSES, AND CAUSES OF ACTION (WHETHER IN
 CONTRACT, TORT (INCLUDING NEGLIGENCE) OR OTHERWISE) EXCEED THE AMOUNT OF
 US$10.
 .
 Notice:  The Software is subject to United States export laws and
 regulations.  You agree to comply with all domestic and international
 export laws and regulations that apply to the Software, including but
 not limited to the Export Administration Regulations administered by the
 U.S. Department of Commerce and International Traffic in Arm Regulations
 administered by the U.S. Department of State.  These laws include
 restrictions on destinations, end users and end use.
