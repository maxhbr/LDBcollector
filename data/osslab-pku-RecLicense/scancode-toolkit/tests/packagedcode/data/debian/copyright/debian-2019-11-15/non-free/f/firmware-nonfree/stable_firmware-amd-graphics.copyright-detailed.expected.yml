primary_license: None
declared_license:
  - Binary redistribution (AMD permissive)
  - Binary redistribution (AMD restrictive)
license_expression: mit AND amd-linux-firmware
copyright: |
  2000-2009, Advanced Micro Devices, Inc.
  2009-2016, Advanced Micro Devices, Inc.
matches:
  - score: '97.67'
    start_line: 14
    end_line: 31
    matcher: 3-seq
    rule_length: 168
    matched_length: 168
    match_coverage: '100.0'
    rule_relevance: 100
    identifier: mit_x11-r75_58.RULE
    license_expression: mit
    is_license_text: yes
    is_license_notice: no
    is_license_reference: no
    is_license_tag: no
    is_license_intro: no
    matched_text: |
      Permission is hereby granted, free of charge, to any person obtaining a
      copy of this software and associated documentation files (the "Software"),
      to deal in the Software without restriction, including without limitation
      the rights to use, copy, modify, merge, publish, distribute, sublicense,
      and/or sell copies of the Software, and to permit persons to whom the
      Software is furnished to do so, subject to the following conditions:

      The above copyright notice and this permission notice (including the next
      paragraph) shall be included in all copies or substantial portions of the
      Software.

      THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
      IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
      FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
      [IN] [NO] [EVENT] [SHALL] THE COPYRIGHT OWNER(S) AND/OR ITS SUPPLIERS BE
      LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
      OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
      WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  - score: '100.0'
    start_line: 38
    end_line: 86
    matcher: 2-aho
    rule_length: 427
    matched_length: 427
    match_coverage: '100.0'
    rule_relevance: 100
    identifier: amd-linux-firmware.LICENSE
    license_expression: amd-linux-firmware
    is_license_text: yes
    is_license_notice: no
    is_license_reference: no
    is_license_tag: no
    is_license_intro: no
    matched_text: |
      REDISTRIBUTION: Permission is hereby granted, free of any license fees,
      to any person obtaining a copy of this microcode (the "Software"), to
      install, reproduce, copy and distribute copies, in binary form only, of
      the Software and to permit persons to whom the Software is provided to
      do the same, provided that the following conditions are met:

      No reverse engineering, decompilation, or disassembly of this Software
      is permitted.

      Redistributions must reproduce the above copyright notice, this
      permission notice, and the following disclaimers and notices in the
      Software documentation and/or other materials provided with the
      Software.

      DISCLAIMER: THE USE OF THE SOFTWARE IS AT YOUR SOLE RISK.  THE SOFTWARE
      IS PROVIDED "AS IS" AND WITHOUT WARRANTY OF ANY KIND AND COPYRIGHT
      HOLDER AND ITS LICENSORS EXPRESSLY DISCLAIM ALL WARRANTIES, EXPRESS AND
      IMPLIED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
      MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT.
      COPYRIGHT HOLDER AND ITS LICENSORS DO NOT WARRANT THAT THE SOFTWARE WILL
      MEET YOUR REQUIREMENTS, OR THAT THE OPERATION OF THE SOFTWARE WILL BE
      UNINTERRUPTED OR ERROR-FREE.  THE ENTIRE RISK ASSOCIATED WITH THE USE OF
      THE SOFTWARE IS ASSUMED BY YOU.  FURTHERMORE, COPYRIGHT HOLDER AND ITS
      LICENSORS DO NOT WARRANT OR MAKE ANY REPRESENTATIONS REGARDING THE USE
      OR THE RESULTS OF THE USE OF THE SOFTWARE IN TERMS OF ITS CORRECTNESS,
      ACCURACY, RELIABILITY, CURRENTNESS, OR OTHERWISE.

      DISCLAIMER: UNDER NO CIRCUMSTANCES INCLUDING NEGLIGENCE, SHALL COPYRIGHT
      HOLDER AND ITS LICENSORS OR ITS DIRECTORS, OFFICERS, EMPLOYEES OR AGENTS
      ("AUTHORIZED REPRESENTATIVES") BE LIABLE FOR ANY INCIDENTAL, INDIRECT,
      SPECIAL OR CONSEQUENTIAL DAMAGES (INCLUDING DAMAGES FOR LOSS OF BUSINESS
      PROFITS, BUSINESS INTERRUPTION, LOSS OF BUSINESS INFORMATION, AND THE
      LIKE) ARISING OUT OF THE USE, MISUSE OR INABILITY TO USE THE SOFTWARE,
      BREACH OR DEFAULT, INCLUDING THOSE ARISING FROM INFRINGEMENT OR ALLEGED
      INFRINGEMENT OF ANY PATENT, TRADEMARK, COPYRIGHT OR OTHER INTELLECTUAL
      PROPERTY RIGHT EVEN IF COPYRIGHT HOLDER AND ITS AUTHORIZED
      REPRESENTATIVES HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.  IN
      NO EVENT SHALL COPYRIGHT HOLDER OR ITS AUTHORIZED REPRESENTATIVES TOTAL
      LIABILITY FOR ALL DAMAGES, LOSSES, AND CAUSES OF ACTION (WHETHER IN
      CONTRACT, TORT (INCLUDING NEGLIGENCE) OR OTHERWISE) EXCEED THE AMOUNT OF
      US$10.

      Notice:  The Software is subject to United States export laws and
      regulations.  You agree to comply with all domestic and international
      export laws and regulations that apply to the Software, including but
      not limited to the Export Administration Regulations administered by the
      U.S. Department of Commerce and International Traffic in Arm Regulations
      administered by the U.S. Department of State.  These laws include
      restrictions on destinations, end users and end use.
